docker/docker-compose.yml

```diff
@@ -4,6 +4,18 @@ services:
   gp2gp:
     image: uk.nhs/gp2gp:0.0.1-SNAPSHOT
     ports:
-      - "8080:8080"
+      - "9000:8080"
     environment:
-      - GP2GP_SERVER_PORT=8080
\ No newline at end of file
+      - GP2GP_SERVER_PORT=8080
+      - GP2GP_LOGGING_LEVEL
+      - GP2GP_AMQP_BROKERS=${GP2GP_AMQP_BROKERS}
```

Comment:
I think these should be default values or blank. Just defining the ENV var GP2GP_AMQP_BROKERS would override the default so assigning it to ${GP2GP_AMQP_BROKERS} doesn't actually do anything.